test/server-test.js

```diff
@@ -115,10 +115,12 @@ describe('SPDY Server', function() {
         assert.equal(req.method, 'POST');
         assert.equal(req.url, '/post');
 
-        res.writeHead(200, {
-          ok: 'yes'
-        });
-        res.end('response');
+        setTimeout(function() {
```

Comment:
Not my proudest moment, but it was either this or introducing a nested `process.nextTick()` in `handle.js`, which was uglier. The issue is that by the time the test gets around to reading the incoming data from the client, the response has ended, and the call to stream.end() ends up terminating the whole stream before the `end` event fires on the request object.